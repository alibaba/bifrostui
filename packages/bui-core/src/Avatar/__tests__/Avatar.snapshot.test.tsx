import { snapshotTest } from 'testing';

describe('Avatar snapshot', () => {
  snapshotTest('Avatar');
});
