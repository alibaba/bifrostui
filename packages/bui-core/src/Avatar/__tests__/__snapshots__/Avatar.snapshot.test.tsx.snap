// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar snapshot Avatar demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-avatar-children"
    >
      UC
    </div>
  </div>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-avatar-children"
    >
      UFCEAEF
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-avatar-children"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"m46.4 16 .3-.3c.9-.7 2.1-.5 2.8.3L66 36.8c.6.7 1.5 1 2.4.6l16.1-6.9c.4-.2.8-.2 1.2-.1 1.1.2 1.8 1.3 1.6 2.4l-8.9 42.9c-.8 3.7-4 6.4-7.8 6.4h-45c-3.8 0-7.1-2.7-7.8-6.4l-9-42.9c-.1-.4 0-.8.1-1.2.4-1 1.6-1.5 2.6-1.1l16.1 6.9c.8.4 1.8.1 2.4-.6L46.4 16z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-square"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar bui-avatar-xsmall bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-avatar bui-avatar-small bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-avatar bui-avatar-medium bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-avatar bui-avatar-large bui-avatar-rounded"
  >
    <div
      className="bui-image bui-avatar-img"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <img
        className="bui-image-img bui-image-loaded"
        loading="eager"
        onError={[Function]}
        onLoad={[Function]}
        src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
        style={
          Object {
            "objectFit": "cover",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 1,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 7`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 1,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-square"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-square"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-square"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-square"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 1,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 8`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-extra bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-avatar-children"
      >
        +2
      </div>
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 9`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 1,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 0,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 1,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-undefined",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Avatar snapshot Avatar demo snapshot 0 10`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-avatar-group"
  >
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-10px",
          "zIndex": 4,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-avatar bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-10px",
          "zIndex": 3,
        }
      }
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="https://gw.alicdn.com/tfs/TB1Q2zRXuL2gK0jSZFmXXc7iXXa-120-120.png"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      className="bui-avatar bui-avatar-group-extra bui-avatar-medium bui-avatar-rounded"
      style={
        Object {
          "marginLeft": "-10px",
          "zIndex": 2,
        }
      }
    >
      <div
        className="bui-avatar-children"
      >
        +2
      </div>
    </div>
  </div>
</div>
`;
